src/main/java/org/openjdk/jextract/impl/TreeMaker.java

```diff
@@ -343,7 +360,7 @@ private Declaration.Typedef createTypedef(Cursor c) {
             Type pointeeType = null;
             try {
                 pointeeType = ((Type.Delegated)canonicalType).type();
-            } catch (NullPointerException npe) {
+            } catch (IllegalStateException unresolved) {
```

Comment:
This catch block is never taken according to coverage report. Maybe it can just be removed?